Apps/WebClient/src/ClientApp/src/components/timeline/note.vue

```diff
@@ -238,6 +240,7 @@ export default class NoteTimelineComponent extends Vue {
     private dateString: string = new Date().toISOString().slice(0, 10);
     private detailsVisible = false;
     private hasErrors: boolean = false;
```

Comment:
Naming suggestion: hasErrors --> isError

Reply:
idk, I think I like hasError better since is error implies this component is an error

Reply:
sounds good to me too